### PR TITLE
Feature/16-scripts-for-importing-adas-bms-data-to-ascot into develop

### PR DESCRIPTION
**Pull request by patrikollus**
Merge branch feature/16-scripts-for-importing-adas-bms-data-to-ascot into branch develop.

### CODE REVIEW

File: a5py/templates/importdata.py [hunk 1]
```diff
@@ -252,7 +332,7 @@ def import_geqdsk(self, fn="input.eqdsk", cocos=None, phiclockwise=None,
                                 right=eqd["fpol"][-1])
         else:
             fpolrz  = np.interp(eqd["psi"], psigrid[::-1], eqd["fpol"][::-1],
-                                right=eqd["fpol"][-1])
+                                riOAght=eqd["fpol"][-1])
```
Comment (miekkasarki) [inline a5py/templates/importdata.py:335]: This is accidental

**Review by miekkasarki — CHANGES_REQUESTED** [review]

File: a5py/templates/importdata.py [hunk 2]
```diff
@@ -1,30 +1,71 @@
...
+        mltpresdens : int
+            Resolution multiplier for density abscissa for BMS coefficients.
+        mltprestemp : int
+            Resolution multiplier for temperature abscissa for
+            BMS coefficients.
+        show_progress : boolean
+            Flag to determine if progress in converting cross-sections into
+            rate coefficients should be printed to terminal.
+        **kwargs
+            Parameters passed to the template.
```
[the remark this answers is not in the document]
Comment (patrikollus) [reply a5py/templates/importdata.py:68]: How's this?
"
Parameters in the form of key-value pairs passed to the template. The key names, e.g. input_CX_H1H0 or input_BMS_H0H1, must follow a specific format where the reaction type (CX or BMS), the reactant species (H and H or H and H) and charge states (1 and 0 or 0 and 1) are specified. For the species and charge, the order is: fast particle first, bulk particle second. The values of the key-value pairs are the full paths to the relevant ADAS reaction data files in the form of strings.
"

File: a5py/templates/importdata.py [hunk 3]
```diff
@@ -1,30 +1,71 @@
...
+#     mltpresekinbms=1, mltpresdensbms=4, mltprestempbms=128,
+#     input_CX_H1H0='/home/adas/adas/adf24/scx#h0/scx#h0_ornl#h1.dat',
+#     input_BMS_H0H1='/home/adas/adas/adf21/bms10#h/bms10#h_h1.dat')
+
 import os
 import numpy as np
 import warnings
-from freeqdsk import geqdsk
-from scipy.interpolate import RectBivariateSpline
+#from freeqdsk import geqdsk
```
[the remark this answers is not in the document]
Comment (miekkasarki) [reply a5py/templates/importdata.py:4]: @patrikollus This line is still commented, right?

[the remark this answers is not in the document]
Comment (patrikollus) [reply a5py/templates/importdata.py]: @miekkasarki Darn it, yes it was. I have to do that to run on Freia, and I forgot to undo it. Now fixed and recommitted.

File: a5py/templates/importdata.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (miekkasarki) [reply a5py/templates/importdata.py]: More explicit like this:
```
**kwargs
    ADAS data files in format: ``reaction``:"path_to_reaction_data"`.
    
    The key ```reaction`` is used to interpret the specific reaction and reactant species the data corresponds to, and it must follow the format ``"input_"<reaction>_<fast particle species><bulk particle species>``. Examples of valid key-value pairs are are ``input_CX_H1H0:"adas/FILE"`` and ``input_BMS_H1H0:"adas/FILE"``.
```
Add line changes and on those replaces those two FILEs with names that actually are used in ADAS database (which you use).

**Review by miekkasarki — APPROVED** [review]

**Pull request merged**